psga/common/primer_autodetection.nf

```diff
@@ -2,7 +2,7 @@
  * Run: primer-autodetection (use first read only for illumina)
  */
 process primer_autodetection {
-  publishDir "${params.output_path}/primer_autodetection", mode: 'copy', overwrite: true, pattern: '{*_primer_data.csv,*_primer_detection.csv}'
+  publishDir "${params.output_path}/primer_autodetection", mode: 'copy', overwrite: true, pattern: '{*_primer_data.csv,*_primer_detection.csv,*_trimming_report.txt}'
```

Comment:
This nextflow file has critical changes:
* run trimmer tool
* add `--primer-base-window` argument to primer autodetection script.